data/glossary.yaml

```diff
@@ -0,0 +1,158 @@
+A2A:
```

Comment:
is the long field being used in these glossary entries? or could it be used for the glossary page instead of duplicating content in both places?